README.md

```diff
@@ -1,24 +1,76 @@
-# AAPB-CLAMS annotation repository 
+[CLAMS.ai](https://clams.ai/) team   
+Author: Jeremy Huey, Data Annotation Manager @jarumihooi  
+Project Manager: Keigh Rim @keighrim  
+aapb-annotations-readme version b.b  
+08/28/2023
```

Comment:
Manually added bylines are technical debt for future maintenance. I suggest we don't use manual bylines.